### DISCUSSION TITLE
4/17 (월) 성하 - @ResponseBody VS ResponseEntity<T>

### DISCUSSION THREAD

**Discussion by sh111-coder** [woowacourse-study/2023-presentation-study#4] [발표 피드백]
[발표](https://ksh-coding.tistory.com/89)

**Comment by kwonyj1022** [comment]
* 설명이 간결하고 중간중간 `~하겠죠?` 이런식으로 소통하면서 발표해서 이해하기 좋았습니다.
* 예제 코드가 있어서 좋았다.

**Comment by jyeost** [comment]
- 역시 성하 👍🏻
- 너무 카컨만 하시는 건 아닌지... 일타강사도 좋지만 대중들도 챙겨주시길 ㅎㅅㅎ

**Comment by amaran-th** [comment]
- 언제나처럼 발표 자료가 깔끔해서 좋았습니다! 가독성 최고
- 딱히 모난 점 없이 무난무난했습니당 지적할 게 없네요

**Comment by JJ503** [comment]
- 오늘도 깔끔한 발표 자료와 발표 실력 👍
- 예제 코드라든지 강조할 부분들에 표시해 두는 것이 보기 좋았습니다.

**Comment by yeonkkk** [comment]
* 제가 성하 카감이라서 일타강사 느낌으로 카메라에 집중하셔서 좋았습니다. 하지만 청자들을 보고 해도 좋을 것 같아요 ㅎ 
* 말하는 속도와 자료의 분량이 적절했어요!
* 예제 코드가 있어서 이해하기 좋았습니다.
